### PR TITLE
net-fs/nfs-utils: import from portage-stable

### PR DESCRIPTION
**Pull request by mischief**

### CODE REVIEW

**Comment by crawford** [comment]
Can you prune the unused patches from "files/"?

**Comment by crawford** [comment]
Woops, nevermind.

**Comment by crawford** [comment]
We don't need the initd files.

**Comment by mischief** [comment]
@crawford: although i did remove the unused patches, removing the initd files will require modification of the ebuild. since this is the initial import from portage-stable, ok to do that in another commit?

**Comment by crawford** [comment]
@mischief That's fair. Can you add another commit that prunes the unneeded files?

**Comment by mischief** [comment]
woops.

**Comment by mischief** [comment]
PTAL

**Comment by mischief** [comment]
@marineam can nfs.confd and nfsmount.confd be nuked, if we only use the newest revs of the service files? according to coreos/baselayout#14 confd files are no longer needed.

**Comment by marineam** [comment]
Yes, conf.d should not be used. If the version we were shipping still used them but a newer ebuild version has switched then we should use the newer revision. If there are any lingering references they should be pruned

**Comment by mischief** [comment]
@marineam, ok, done. PTAL

File: net-fs/nfs-utils/files/nfs-utils.conf [hunk 1]
```diff
@@ -0,0 +1,3 @@
+d /var/lib/nfs/rpc_pipefs
```
Comment (marineam) [inline net-fs/nfs-utils/files/nfs-utils.conf]: Is this sufficient for tmpfiles? I always write them with the `-` place holder for default/unused fields.

**Comment by mischief** [comment]
that's how it was in the upstream package. https://github.com/coreos/portage-stable/blob/52dfc138f545f29971a4fcab2fe6794119b20925/net-fs/nfs-utils/files/nfs-utils.conf

i can make it explicit if that's better.

**Comment by marineam** [comment]
Ok, that's fine then. LGTM

**Pull request merged**